curses/CMakeLists.txt

```diff
@@ -3,6 +3,10 @@ project(tn5250-curses LANGUAGES C)
 
 include_directories(${CMAKE_BINARY_DIR} ../lib5250)
 
+if (NOT CURSES_OLD_KEYS)
+    target_compile_definitions(tn5250 PUBLIC -D USE_OWN_KEY_PARSING=1)
+endif()
+
```

Comment:
I think this has to go after the target is defined, no? Either way I think it makes logical sense to add it either before or after `target_link_libraries` below.